### PR TITLE
Allow lumberdash to accept a list of clients

### PR DESCRIPTION
**Pull request by jorgecoca**

### CODE REVIEW

File: lib/src/lumberdash.dart [hunk 1]
```diff
@@ -2,46 +2,43 @@ import 'package:meta/meta.dart';
...
+  assert(withClients != null);
+  _lumberdashClients = withClients;
 }
 
-/// It calls the `logMessage` method of the given [LumberdashClient]
+/// It calls the `logMessage` method of the each [LumberdashClient]
 /// passed to `putLumberdashToWork`, and returns the given `message`.
 String logMessage(String message, [Map<String, dynamic> extras]) {
-  _lumberdashClient.logMessage(message, extras);
+  // _lumberdashClient.logMessage(message, extras);
```
Comment (felangel) [inline lib/src/lumberdash.dart]:
```suggestion
```

File: plugins/colorize_lumberdash/pubspec.yaml [hunk 2]
```diff
@@ -10,7 +10,8 @@ environment:
   sdk: ">=2.4.0 <3.0.0"
 
 dependencies:
-  lumberdash: ^2.0.0
+  lumberdash:
```
Comment (felangel) [inline plugins/colorize_lumberdash/pubspec.yaml]: shouldn't we revert this?

File: plugins/colorize_lumberdash/example/pubspec.yaml [hunk 3]
```diff
@@ -3,12 +3,12 @@ description: Example project for Lumberjack
 version: 1.0.0
 
 environment:
-  sdk: ">=2.0.0 <3.0.0"
+  sdk: ">=2.4.0 <3.0.0"
 
 dependencies:
-  lumberdash: ^1.0.0
+  lumberdash:
```
Comment (felangel) [inline plugins/colorize_lumberdash/example/pubspec.yaml]: shouldn't we revert this?

File: plugins/firebase_lumberdash/example/pubspec.yaml [hunk 4]
```diff
@@ -8,7 +8,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  lumberdash: ^1.0.0
+  lumberdash:
```
Comment (felangel) [inline plugins/firebase_lumberdash/example/pubspec.yaml]: shouldn't we revert this?

File: plugins/firebase_lumberdash/pubspec.yaml [hunk 5]
```diff
@@ -13,7 +13,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  lumberdash: ^1.0.0
+  lumberdash:
```
Comment (felangel) [inline plugins/firebase_lumberdash/pubspec.yaml]: shouldn't we revert this?

**Review by felangel — APPROVED** [review]
LGTM 👍

**Review by tchabot22 — APPROVED** [review]

**Review by erickjtorres — APPROVED** [review]

**Pull request merged**